README.md

```diff
@@ -1,114 +1,143 @@
-
-    sudo apt-get purge nodejs npm
-    ## Pi2 | wget https://nodejs.org/dist/v6.9.3/node-v6.9.3-linux-armv7l.tar.xz -O node.tar.xz
-    ## Pi A/A+, B/B+ und Zero (ARMv6) | wget https://nodejs.org/dist/v6.9.3/node-v6.9.3-linux-armv6l.tar.xz -O node.tar.xz
-    tar -xvf node.tar.xz
-    cd node-v6.9.3-linux-armv*
-
-## Installation with Docker
-
-*This version is currently outdated.*
-
-[itzg](https://hub.docker.com/r/itzg/) has made a great docker container for rcon web admin. If you prefer docker, you can do it with https://hub.docker.com/r/itzg/rcon/
-    
-## Start/Stop on Linux
-
-    sh startscripts/start-linux.sh start
-    sh startscripts/start-linux.sh stop
-    sh startscripts/start-linux.sh restart
-    
-## Start/Stop on Windows - Close cmd window to close
-
-    startscripts/start-windows.bat
-    
-## Open in browser
-Goto: http://yourserverip:4326 (You can also use your hostname instead of ip).
-To modify the :4326 port or allowed hosts, have a look in the `config.template.js` file in the root folder.
-
-## Boot scripts
-
-On linux you can start the rcon web admin with your server start. For example on ubuntu you can simply add a `crontab -e` line. Do this with the user you want to start the script with, not `sudo`.
-    
-    @reboot /path/to/startscripts/start-linux.sh start
-
-## Widget developers
-
-Goto https://github.com/lacaulac/rcon-web-admin/tree/master/public/widgets for more information.
+
+* NodeJS v12.13.1
+
+## Windows Installation
+
+* Download and install [NodeJS](https://nodejs.org)
+* Download and unpack this repository
+
+In the extracted directory, run the following commands:
+```bash
+$ npm install
+$ node src/main.js install-core-widgets
+```
+
+## Linux Installation
+
+> **Note**: It is strongly advised to avoid running this application as root or with `sudo`, it is not required.
+> Also, avoid installing this application in a directory which can be accessed from the internet.
+> This application launches a self-contained webserver with limited access to the `/public` folder.
+> Failure to follow this advice could leave you vulnerable to security risks.
+
+Run the following commands in the shell:
+```bash
+$ sudo apt-get install nodejs npm
+$ sudo npm update npm -g
+$ wget https://codeload.github.com/lacaulac/rcon-web-admin/zip/master -O rcon-web-admin.zip
+$ unzip rcon-web-admin.zip
+$ mv rcon-web-admin-master rcon-web-admin
+$ cd rcon-web-admin
+$ npm install
+$ node src/main.js install-core-widgets
+$ chmod 0755 -R startscripts *
+```
+
+## Environment Variables
+
+RCON Web Admin can be partially configured with environment variables. You may configure the initial user and server with these environment variables.
+
+| Variable | Description | Accepted values | Default value |
+|:--------:|:-----------:|:---------------:|:-------------:|
+| RWA_ENV | Enables configuration by environment variables | TRUE / FALSE | FALSE |
```

Comment:
I have pre-enabled this setting here

https://github.com/itzg/docker-rcon-web-admin/blob/master/Dockerfile#L23

since environment variable driven configuration will be perfect for containers. When this PR is merged I will link to this README section from the [Options](https://github.com/itzg/docker-rcon-web-admin#options) section of the image.